api-testing/functional/tests/cypress/fixtures/bodyData.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
It represents the body data for an API for the institution update from an Institution Admin user and the file is named just as `bodyData`. I assume there is a plan to have it better organized, right?

Reply:
Will change this file name into institutionUpdateData.

Reply:
Changes have been made & file name changed into institutionUpdateData.